lib/handlerTemplate.js

```diff
@@ -444,7 +459,16 @@ function createHandler(spec, options) {
             _doReturn: false,
             _spec: spec,
         };
-        Object.assign(ctx.model, options.globals);
+        if (modelInit) {
+            var model = ctx.model;
+            // Don't use Object.assign, as we want to give precedence to the
+            // model.
+            Object.keys(modelInit).forEach(function(key) {
```

Comment:
The difference is that one always copies everything, while the other typically does nothing.